src/pkgcheck/checks/codingstyle.py

```diff
@@ -903,3 +903,102 @@ def feed(self, pkg):
                         yield RedundantDodir(
                             cmd.group('cmd'), line=dodir.group('call'),
                             lineno=lineno - 1, pkg=pkg)
+
+
+class EbuildUnquotedVariable(results.LineResult, results.Style):
```

Comment:
Style level results imply that changes are optional and up to the opinion of the user. I'd recommend using at least warning level here or even error if you handle or skip the false positives.

Reply:
Technically some function calls will still report "unnecessary" positives like bash's append-cppflags (it doesn't need quotes, similarly to the whitelisted echo/elog/etc... given append-* just takes $* and add it to cppflags). But that kinda assume how eclass functions work and feel may become an issue down the line.

So I'd say these should be fixed too, so warning level may not sound too bad.

Reply:
Changed to warning, thanks for the input!